scripts/wca-bootstrap.sh

```diff
@@ -63,17 +61,17 @@ EOL
   fi
   # Let the root user do the git-ing, and then chown appropriately.
   if ! [ -d $repo_root ]; then
-    git clone -b $git_branch --recursive https://github.com/cubing/worldcubeassociation.org.git $repo_root
+    git clone -b $git_branch https://github.com/cubing/worldcubeassociation.org.git $repo_root
```

Comment:
Woohoo! I believe that now that we're submodule-less, we can get rid of the "initialize submodules" step in our README? https://github.com/cubing/worldcubeassociation.org#setup